lib/themes.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+
+final ThemeData appTheme = ThemeData(
+  primaryColor: Colors.white,
```

Comment:
Should we change cursor color in TextFormField as well? 🙏 
<img src="https://user-images.githubusercontent.com/60863885/112285104-864f0d00-8cbc-11eb-8053-f68ef4858a28.jpg" width=350 />

Reply:
👌